frontend/src/components/design-system/containers/PillComponent.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+export enum ColorPill {
```

Comment:
Zatím to pojďme držet beze enumů, tj. ty hodnoty rovnou v Props intefacu.

Reply:
Dobře, smazala jsem.